x/staking/spec/01_state.md

```diff
@@ -43,6 +44,11 @@ Validators can have one of three statuses
   tombstoning, an unbonding of all their delegations begins. All delegations must then wait the UnbondingTime
   before moving receiving their tokens to their accounts from the `BondedPool`.
 
+Thoughts on validator status update: 
+- `Unbonded` status is set at initial validator creation as it is.
```

Comment:
Should this also be set at the epoching process?  Buffer the "CreateValidator" msg as well?